lib/ptable.h

```diff
@@ -30,17 +32,18 @@
  *  NOTE: data in partition table should be stored in little endian
  */
 
-/* clang-format off */
 
 /* Partition table magic signature */
 static const u8 ptable_magic[] = { 0xde, 0xad, 0xfc, 0xbe };
 
 
 /* Supported partition types */
-enum { ptable_raw = 0x51, ptable_jffs2 = 0x72, ptable_meterfs = 0x75 };
-
+/* clang-format off */
+enum { ptable_raw = 0x51, ptable_jffs2 = 0x72, ptable_meterfs = 0x75, ptable_futurefs = 0x78 };
```

Comment:
IMHO better to not define `futurefs` - when we decide to define it eventually, it would require API breaking change